test/lib/commands/link.js

```diff
@@ -315,14 +315,13 @@ t.test('link pkg already in global space', async t => {
   })
 
   // XXX: how to convert this to a config that gets passed in?
-  npm.config.find = () => 'default'
+  npm.config.find = () => '--save'
```

Comment:
This was an existing test that was testing something in a global context, it's not appropriate to change it to ALSO try to cover this use case.  A net new test will be needed.

Reply:
Added new test case.